src/main/java/org/EmailCarbonFootprint/GmailAccountDetails.java

```diff
@@ -7,44 +7,45 @@
 public class GmailAccountDetails {
 
 	private String emailId;
-	private String appPassword;
-	
+	private String gmailAppPassword;
```

Comment:
"gmailAppPassword" variable name can be more specific like "emailIdPassword" or "gmailIdPassword".

Reply:
It is not gmail id password. It is app password which is generated automatically.